js/impress.js

```diff
@@ -133,11 +133,6 @@
         return " scale(" + s + ") ";
     };
 
-    // `perspective` builds a perspective transform string for given data.
-    var perspective = function( p ) {
-        return " perspective(" + p + "px) ";
-    };
-
```

Comment:
I agree it's a good thing remove the function here since it doesn't have good cohesion (`perspective()` says nothing really). Sometimes no abstraction is better than the wrong abstraction 👍